user_guide_src/source/changelogs/v4.6.0.rst

```diff
@@ -75,6 +75,9 @@ Helpers and Functions
 
 Others
 ======
+- **Services:** Added ``BaseService::updateServicesCache`` method to allow
+  updating the services cache file post autoloading. This will only look for
+  new services and add them to the class name cache.
```

Comment:
```suggestion
  updating the cache (property) of services created when Services is first called.
  This will only look for new Services classes and add the classnames to the cache.
```